src/cmd/prepare.go

```diff
@@ -205,6 +206,29 @@ var prepareGenerateConfigFile = &cobra.Command{
 	},
 }
 
+var lintCmd = &cobra.Command{
+	Use:     "lint [ DIRECTORY ]",
+	Args:    cobra.MaximumNArgs(1),
+	Aliases: []string{"l"},
+	Short:   lang.CmdLintShort,
+	Run: func(cmd *cobra.Command, args []string) {
+		baseDir := ""
```

Comment:
I was initially hesitant to use pkg / creatopts because in order for the lint to work I need a untyped unmarshalled yaml variable. However, I ended up also needing a typed variable when I checked for ###ZARF_PKG_TMPL_ anyway. I will try out using createOpts again

Reply:
Ended up using createopts and moved lint code to packager